tests/sentry/projects/project_rules/test_creator.py

```diff
@@ -36,8 +36,8 @@ def setUp(self):
         )
 
     def test_creates_rule(self):
-        r = self.creator.call()
-        rule = Rule.objects.get(id=r.id)
+        r = self.creator.run()
+        rule: Rule = Rule.objects.get(id=r.id)
```

Comment:
You shouldn't need to provide a type unless there is another variable that is shadowing `rule`.